Pyrado/pyrado/sampling/parallel_rollout_sampler.py

```diff
@@ -256,29 +250,29 @@ def sample(
                     # Run every initial state so often that we at least get min_rollouts trajectories
                     func = partial(_ps_run_one_init_state, eval=eval)
                     rep_factor = ceil(self.min_rollouts / len(init_states))
-                    arglist = rep_factor * init_states
+                    arglist = list(product(range(rep_factor), init_states))
```

Comment:
Do the two statements yield the same? If so, why did you change it?

Reply:
They yield the same number of rollouts, but the changed statement includes the rollout number in a tuple. But damn, this is exactly why I included the initial state in the seed at first: assuming the initial state is irrelevant for the rollout for some environment, this would sample the same rollout multiple times. But this was also the case if the same initial state was passed twice. I'll change this to give each rollout an own number.

Reply:
I changed it to `enumerate(rep_factor * init_states)` to give each rollout to be sampled a unique ID. The `list(…)` is necessary because `run_map` checks the `len()` and `enumerate` has no `len()`.